src/main/java/net/snowflake/ingest/streaming/OpenChannelRequest.java

```diff
@@ -161,10 +157,6 @@ public String getOffsetToken() {
     return this.offsetToken;
   }
```

Comment:
1. please undo these changes. you're changing public API surface of a released SDK.

2. @sfc-gh-tzhang do we have CI tools to catch changes to public API surface? If not lets add a backlog item to catch accidental API surface area changes before a release?

Reply:
Undo changes.